qualtran/_infra/composite_bloq.py

```diff
@@ -62,6 +65,16 @@
 """
 
 
+def _to_tuple(x: Iterable[Connection]) -> Sequence[Connection]:
+    """Needed so mypy can correctly infer types."""
```

Comment:
"""mypy-compatible attrs converter for CompositeBloq.connections"""

^ suggestion

Reply:
Done.